test/integration/keyboard.integration.ts

```diff
@@ -0,0 +1,51 @@
+import { test, expect } from '@playwright/test';
+
+import senseHorizon from '../rendering/theme/sense-horizon';
+import NebulaFixture from './utils/nebula-fixture';
+
+import SnTable from './utils/sn-table';
+import { expectations } from './utils/expectations';
+import { navigateWithArrowsMsg } from './utils/custom-error-messages';
+
+test.describe('Fixture rendering', () => {
```

Comment:
`Fixture rendering`?

Reply:
A test file can contain different test cases for different environments. 
This described block will contain only those test cases served by Nebula with a fixture file. Other describe blocks will contain the Backend or the Enigma mocker served test cases.
If the title is not too informative, I can change it to 'Tests served by: fixture-file rendering by Nebula'.